### ISSUE TITLE
append-query has a dependency containing a critical vulnerability

### ISSUE THREAD

**Issue by andyjdavis** [lakenen/node-append-query#10]
Retire (https://www.npmjs.com/package/retire) output
```
node_modules/append-query/node_modules/extend/package.json
 ↳ extend 2.0.2
extend 2.0.2 has known vulnerabilities: severity: critical; summary: Prototype pollution attack; https://hackerone.com/reports/381185
```
Bumping the version of extend should be sufficient to resolve this.

**Comment by andyjdavis** [comment]
Here is a PR that should resolve this. https://github.com/lakenen/node-append-query/pull/11

**Comment by lakenen** [comment]
Thanks! Just tagged and published 2.1.1 on npm.
